webcompat/static/js/lib/bugform.js

```diff
@@ -216,6 +219,60 @@ function BugForm() {
     this.submitButtons.removeClass("is-disabled");
   };
 
+  this.showSimilarBugs = function() {
+    if (!$("body").data("username")) {
+      // Not login
+      return;
+    }
+
+    if (this.suggestDelay) {
+      clearTimeout(this.suggestDelay);
+    }
+    var bugform = this;
+    this.suggestDelay = setTimeout(function() {
+      try {
+        var url = new URL(bugform.urlField.val());
+        if (url.hostname.length > 7) {
+          var query = `https://api.github.com/search/issues?page=1&per_page=10&stage=all&sort=created&q=${url.hostname}+in:title+repo%3Awebcompat%2Fweb-bugs&order=desc`;
+
+          fetch(query)
```

Comment:
This is fine for a prototype, but we need to be maximally compatible with old browsers for bug reporting:

https://caniuse.com/#feat=fetch

Reply:
+1, so this is only for prototyping, let's discuss this in all hands to know if this really improve bug quality? Any change could make it better?

Reply:
+1